packages/commercetools/composables/src/useForgotPassword/index.ts

```diff
@@ -0,0 +1,28 @@
+import { Context, UseForgotPassword, useForgotPasswordFactory, UseForgotPasswordFactoryParams } from '@vue-storefront/core';
+
+const useForgotPasswordFactoryParams: UseForgotPasswordFactoryParams<any> = {
+  resetPassword: async (context: Context, { email, customQuery }) => {
+    try {
+      return await context.$ct.api.customerCreatePasswordResetToken(email, customQuery);
```

Comment:
I think `await` might be redundant in this case

Reply:
If so, in method below too

Reply:
It was not necessary. It is working without it as well.